src/charm.py

```diff
@@ -533,21 +534,26 @@ def _on_set_password(self, event: ActionEvent) -> None:
         )
 
     def _on_secret_remove(self, event):
-        logging.debug(f"Secret {event._id} seems to have no observers, could be removed")
+        logging.debug(f"Secret {event.secret.id} seems to have no observers, could be removed")
 
     def _on_secret_changed(self, event):
-        secret = event.secret
+        regex = re.compile(".*[^/][/:]")
```

Comment:
I think it is a good idea to add a pattern what this re matches in the comment

Reply:
fixed

Reply:
nits: I mean to write what this re will match, since not everyone good with getting the string out of pattern 😄

Reply:
added